sdk/storage/storage-blob/test/perfstress/track-2/download.spec.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressOptionDictionary } from "@azure/test-utils-perfstress";
+import { StorageBlobTest } from "./storageTest.spec";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+import { BlockBlobClient } from "../../../src";
+import { streamToBuffer3 } from "../../../src/utils/utils.node";
+dotenv.config();
+
+interface StorageBlobDownloadTestOptions {
+  size: number;
+}
+
+export class StorageBlobDownloadTest extends StorageBlobTest<StorageBlobDownloadTestOptions> {
+  public options: PerfStressOptionDictionary<StorageBlobDownloadTestOptions> = {
+    size: {
+      required: true,
+      description: "Size in bytes",
+      shortName: "sz",
+      longName: "size",
+      defaultValue: 1024
+    }
+  };
+
+  static blobName = `newblob${new Date().getTime()}`;
+  blockBlobClient: BlockBlobClient;
+
+  constructor() {
+    super();
+    this.blockBlobClient = this.containerClient.getBlockBlobClient(
+      StorageBlobDownloadTest.blobName
+    );
+  }
+
+  public async globalSetup() {
+    await super.globalSetup();
+
+    // Create a blob
+    await this.blockBlobClient.upload(
+      Buffer.alloc(this.parsedOptions.size.value!),
+      this.parsedOptions.size.value!
+    );
+  }
+
+  async runAsync(): Promise<void> {
+    const downloadResponse = await this.blockBlobClient.download();
+    await streamToBuffer3(downloadResponse.readableStreamBody!);
```

Comment:
Read to null

Reply:
Added it to the framework
https://github.com/Azure/azure-sdk-for-js/pull/13014